api/db/drop.js

```diff
@@ -15,7 +23,7 @@ const userArgs = minimist(process.argv.slice(2), {
 
 const dropInterests = async () => {
```

Comment:
The users and events collections depend on the interests one, so probably adding a prompt for each asking the developer if they want to drop them is a good idea.